### PR TITLE
centralize handleUserMenuAction

### PR DESCRIPTION
**Pull request by sarahentzel**

### CODE REVIEW

**Comment by sarahentzel** [comment]
So this is taken care of in the new UI already?

**Comment by sarahentzel** [comment]
This has been rebased on develop and AppHead used.

File: src/routes/Loading.tsx [hunk 1]
```diff
@@ -120,18 +121,9 @@ export function Loading(props: IProps) {
     localStorage.getItem('newOrg')
   );
   const [savedURL] = useState(localStorage.getItem('url') || '');
-  const [view, setView] = useState('');
+  const [view] = useState('');
```
Comment (gtryus) [inline src/routes/Loading.tsx:124]: If there is no place to setView in this module, do we want to eliminate the variable completely?

Comment (sarahentzel) [reply src/routes/Loading.tsx]: Done.

**Pull request merged**